configure.ac

```diff
@@ -57,7 +57,7 @@ case $host_os in
          # Homebrew where each one is located, then adjust paths accordingly.
          if $BREW list --versions valgrind >/dev/null; then
            valgrind_prefix=$($BREW --prefix valgrind 2>/dev/null)
-           VALGRIND_CPPFLAGS="-I$valgrind_prefix/include"
+           VALGRIND_CPPFLAGS="-nostdinc -isystem $valgrind_prefix/include"
```

Comment:
I guess there's a reason why you also add `-nostdinc` but I don't see it.

Reply:
On macOS x86_64, the /usr/local/include is included by default. Packages been installed via Homebrew have alternative softlinks in it, in addition to $(brew --prefix package)/include.